whitepaper/chapters/blocks.tex

```diff
@@ -74,7 +74,8 @@ \subsection{Block Fields}
 Each successive block must have a timestamp greater than that of the previous blocks because block time is strictly increasing.
 Each network tries to keep the average time between blocks close to \nind{target block time}.
 
-\field{Difficulty} is described in detail in \nemref{sec:blockchain:difficulty}.
+\field{Difficulty} determines how difficult is to harvest a new block, based on previous blocks.
```

Comment:
if possible, i'd prefer if you could reword to not define difficulty in terms of difficult ^^.

Reply:
Changed "difficult" for "hard"